aws-java-sdk-dynamodb/src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/DynamoDBAtomicIncrementor.java

```diff
@@ -0,0 +1,20 @@
+package com.amazonaws.services.dynamodbv2.datamodeling;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines a pattern for annotating fields that are numeric to apply an incremental value to the database via
+ * either calling save, or by calling the mapper method incr which only applies those attributes which intend
+ * to atomically increment for the specified key value.
+ *
+ * Created by matt@mjgreenwood.net on 3/3/17.
+ */
+@DynamoDB
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD, ElementType.METHOD})
+public @interface DynamoDBAtomicIncrementor {
+    String attributeName() default "";
```

Comment:
I had originally implemented the change this way, but it felt too limiting - I had thought let the POJO dictate the value sent to the service.  I can revert if there's a preference push this in that manner, its not material to my use-case